pycode/memilio-epidata/memilio/epidata/getSimulationData.py

```diff
@@ -65,32 +67,34 @@ def get_simulation_data(read_data=dd.defaultDict['read_data'],
     - getDIVIData.get_divi_data
 
     Keyword arguments:
-    @param read_data False [Default] or True. Defines if data is read from file or downloaded.
+    @param split_berlin True or False. Defines if Berlin's disctricts are kept separated or get merged. Default defined in defaultDict.
+    @param read_data True or False. Defines if data is read from file or downloaded.  Default defined in defaultDict.
     @param file_format File format which is used for writing the data. Default defined in defaultDict.
-    @param out_folder Path to folder where data is written in folder out_folder/Germany.
-    @param no_raw True or False [Default]. Defines if unchanged raw data is saved or not.
-    @param end_date [Optional] Date to stop to download data [Default = today].
-    @param impute_dates False [Default] or True. Defines if dates where nothing changed are added.
-    @param make_plot False [Default] or True. Defines if plots are generated with matplotlib.
-    @param moving_average True or False [Default]. Defines if files for 7 day moving average should be created
-    @param split_berlin True [Default] or False. Defines if Berlin counties is fused to just on county.
-    @param start_date [Optional] Date to start to download data [Default = 2020-04-24].
+    @param out_folder Folder where data is written to. Default defined in defaultDict.
+    @param no_raw True or False. Defines if unchanged raw data is saved or not. Default defined in defaultDict.
+    @param start_date Date of first date in dataframe. Default 2020-01-01.
+    @param end_date Date of last date in dataframe. Default defined in defaultDict.
+    @param impute_dates True or False. Defines if values for dates without new information are imputed. Default defined in defaultDict.
+    @param moving_average Integers >=0. Applies an 'moving_average'-days moving average on all time series
+        to smooth out weekend effects.  Default defined in defaultDict.
```

Comment:
If you go to all of this, maybe change the words from "to smooth out weekend effects" to "to smooth out effects of irregular reporting"

Reply:
Done